src/components/notifications/AppSelector/AppSelector.scss

```diff
@@ -131,6 +131,11 @@
     }
 
     &__title {
+      display: -webkit-box;
+      -webkit-line-clamp: 1;
+      -webkit-box-orient: vertical;
```

Comment:
What's compatibility like for this? Also we have  `autoprefixer`, can we take advantage of that here?

Reply:
As I see the `autoprefixer` didn't add support for `line-clamp` and `box-orient` yet. It's mentioned in a few issues but I don't see a new release after `10.4.14` which includes those changes. Let's keep using the prefixes here.